### PR TITLE
Add TSC meeting notes from 09/28 and 10/05

### PR DESCRIPTION
**Pull request by michdolan**
Signed-off-by: Michael Dolan <michdolan@gmail.com>

### CODE REVIEW

File: ASWF/meetings/tsc/2020-10-05.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+          some general CMake changes and removal of auto-install of 
+          documentation-specific dependencies.
+        - Patrick: Noticed this morning on tsc-private topic related to 
+          copyright of images found with license scan. Needs a look.
+    - #1159 - Improve CMake files:
+        - Patrick: Not C++. Faced issues with CMake on Windows and tried to 
+          fix.
+    - #1157 - FileTransform interpolation:
+        - Patrick: Last meeting discussed v1 not throwing if interpolation was 
+          missing. Bernard checked and it is throwing. Explanation in pull
```
Comment (hodoulp) [inline ASWF/meetings/tsc/2020-10-05.md:44]: I mentioned that `Bernard checked and it is throwing for FileTransform (i.e. LUTs) as expected.`

File: ASWF/meetings/tsc/2020-10-05.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+      much. Two or three things to follow up on though. Worried our concerns 
+      will be lost because of imposed deadline. Would be a shame if features 
+      go out if solutions doesn't match spirit of previous library version.
+    - Mark T: What things are you concerned about?
+    - Mark B: ICC display transforms. Live grading controls and how those 
+      interact. Environment resolution. Bit-depth - how it's expressed in graph 
+      and config.
+    - Patrick: Suggestion - build a list and next TSC meeting would like to 
+      discuss these points. We can address questions over several meetings. 
+      Could explain what we wanted to do and follow up on any concerns.
```
Comment (hodoulp) [inline ASWF/meetings/tsc/2020-10-05.md:100]: Just to better explain my live comment: `Even if we did some WG sessions and created defects for all the features, we could still explain again what we all agreed to do and follow up on any concerns.`

**Review by hodoulp — CHANGES_REQUESTED** [review]

File: ASWF/meetings/tsc/2020-10-05.md (hunk 1 above)
Comment (michdolan) [reply ASWF/meetings/tsc/2020-10-05.md]: Updated.

File: ASWF/meetings/tsc/2020-10-05.md (hunk 2 above)
Comment (michdolan) [reply ASWF/meetings/tsc/2020-10-05.md]: Updated.

**Review by hodoulp — APPROVED** [review]

File: ASWF/meetings/tsc/2020-09-28.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+        - Patrick: Yes, wrote own implementation. Benefit in performance is so 
+          important that both approaches are supported.
+        - Doug: Uses fewer iterations than needed for full float 32-bit results.
+        - Mark: Had similar challenge getting this working in CTL expression 
+          code. Finding concise method of operations challenging. Since we 
+          already have concept of optimizations sounds fine. Is this default?
+        - Doug: Yes, both are turned on by default, in OPTIMIZATION_VERY_GOOD.
+        - Patrick: optimization is based on bits, so can disable any of the 
+          items. Fine control of optimization. There was a question from Zach 
+          Lewis about v1. You can still make v1 config by setting version.
```
Comment (zachlewis) [inline ASWF/meetings/tsc/2020-09-28.md:72]: 👍
Regarding optimization flags and versions, I was curious to know if you guys had given any consideration to format / version compatibility optimization flags / filters...? I.e., something like an OPTIMIZATION_COMPAT_OCIO_1_1 option that would constrain an optimized processor to a subset of ostensibly backward-compatible transforms; or OPTIMIZATION_COMPAT_CTF_1_3 for Flame-2019(?)+ compatible-transforms (if that's even possible); and so on, for future versions of CTF / CLF / OCIO / etc.

(Not a feature request, just curious).

File: ASWF/meetings/tsc/2020-09-28.md [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+          DEFAULT interpolation as default, instead of UNKNOWN, which prevents 
+          exceptions being thrown in various places. Also cleaned up some file 
+          transform interpolation. If you specify an interpolation that can't 
+          be used for file, will log warning. For CLF, where interpolation can 
+          be specified in file, if an op doesn't specify interpolation, it will 
+          try to use the FileTransform interpolation specified. Will ignore 
+          file transform if specified in file. Also addresses issue with 
+          caching. If multiple transforms used specify different 
+          interpolations, that will now work.
+        - Bernard: Also added CAT transform function.
```
Comment (zachlewis) [inline ASWF/meetings/tsc/2020-09-28.md:84]: ?
Are we talking Chromatic Adaptation Transforms in the context of File Transform interpolation?

**Review by zachlewis — APPROVED** [review]
Magnificent notes.

File: ASWF/meetings/tsc/2020-09-28.md [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+          When create new config, it is now v2 by default instead of v1. Other 
+          thing in PR enhances optimization flags. Now ability to control types 
+          of optimization. Two new flags: one simplifies ops down to component 
+          parts; CDL without power becomes just matrix if possible, etc. 
+          Benefit is that can be combined with another matrix op. Another flag 
+          controls whether SSE log and power are used. Does much faster version 
+          of these functions but not 32-bit exact - 16-bit half precision. 
+          Useful to turn it on and off, and off for unit tests since we’re 
+          comparing with higher precision results.
+        - Bernard: Fast log is also used for baking.
```
Comment (doug-walker) [inline ASWF/meetings/tsc/2020-09-28.md:60]: I think Bernard said the opposite.  Suggest replacing with:

Bernard: Fast log is not used for baking.

File: ASWF/meetings/tsc/2020-09-28.md (hunk 3 above)
Comment (doug-walker) [reply ASWF/meetings/tsc/2020-09-28.md:72]: We have not implemented that.

File: ASWF/meetings/tsc/2020-09-28.md (hunk 4 above)
Comment (doug-walker) [reply ASWF/meetings/tsc/2020-09-28.md:84]: There is nothing involving chromatic adaptation in this PR.  I think Bernard's comment was something else.  Suggest:

Bernard: Also added a function to get the type of a transform (CDL, Matrix, etc.).

**Review by doug-walker — CHANGES_REQUESTED** [review]
As usual, these are amazing notes Michael, thank you so much!

Just two items I would suggest to correct so as not to cause confusion.

File: ASWF/meetings/tsc/2020-09-28.md (hunk 4 above)
Comment (michdolan) [reply ASWF/meetings/tsc/2020-09-28.md]: Corrected this based on Doug's suggestion

File: ASWF/meetings/tsc/2020-09-28.md (hunk 5 above)
Comment (michdolan) [reply ASWF/meetings/tsc/2020-09-28.md]: Thanks, fixed.

**Review by zachlewis — APPROVED** [review]

**Review by doug-walker — APPROVED** [review]

**Pull request merged**